StaticAnalyzer/models.py

```diff
@@ -2,20 +2,20 @@
 # Create your models here.
 
 class RecentScansDB(models.Model):
-    NAME = models.TextField()
+    NAME = models.CharField(max_length=200)
     MD5 = models.CharField(max_length=32)
-    URL = models.TextField()
+    URL = models.URLField()
     TS = models.DateTimeField()
 
 class StaticAnalyzerAndroid(models.Model):
-    TITLE = models.TextField()
-    APP_NAME = models.TextField()
+    TITLE = models.CharField(max_length=50)
+    APP_NAME = models.CharField(max_length=150)
     SIZE = models.CharField(max_length=50)
     MD5 = models.CharField(max_length=32)
-    SHA1 = models.TextField()
-    SHA256 = models.TextField()
-    PACKAGENAME = models.TextField()
-    MAINACTIVITY = models.TextField()
+    SHA1 = models.CharField(max_length=40)
+    SHA256 = models.CharField(max_length=64)
+    PACKAGENAME = models.CharField(max_length=150)
+    MAINACTIVITY = models.CharField(max_length=150)

```

Comment:
And here

Reply:
Packagename and mainactivity I have increased the length, if you feel inappropriate, change to Textastic